code/style.css

```diff
@@ -1,4 +1,222 @@
+*{
+  margin: 0;
+  padding: 0;
+}
+*, *::before, *::after{
+  box-sizing:border-box;
+}
+
 html {
-  background: #0025ff;
+  background: #F2002A;
   color: #fff;
+  font-family: 'Carter One', cursive;
+  font-family: 'Roboto', sans-serif;
+  font-size: 12px;
+}
+
+body{
+  min-width: 320px;
+}
+
+h1{
+  font-family: 'Carter One', cursive;
+  font-size: 16px;
+}
+
+h2{
+  padding-top: 10px;
+  font-family: 'Carter One', cursive;
+  font-size: 14px;
+}
+
+h1.title-text{
+  font-family: 'Carter One', cursive;
+  font-size: 30px;
+}
+.header{
+  width: 100%;
+  background-color: white;
+  height: 100px;
+  display: flex;
+  flex-wrap: wrap;
+  align-items: center;
+  justify-content: flex-start;
```

Comment:
Isn't that the default value in regard to the justify-content?

Reply:
flex-start is default, yes.